invenio_cli/commands/assets.py

```diff
@@ -135,26 +163,40 @@ def link_js_module(self, path):
         steps = [
             FunctionStep(  # Install dependent packages
                 func=self._npm_install_command,
-                args={"path": path},
+                args={"path": path, "module_pkg": module_pkg},
                 message="Installing dependent packages...",
             ),
             FunctionStep(  # Run build script
                 func=self._build_script,
                 args={"module_pkg": module_pkg},
                 message="Building...",
             ),
-            FunctionStep(  # Create link to global folder
-                func=self._run_script,
-                args={"module_pkg": module_pkg},
-                message="Linking module to global dist...",
-            ),
-            FunctionStep(  # Link the global folder to the assets folder.
-                func=self._assets_link,
-                args={"assets_pkg": assets_pkg, "module_pkg": module_pkg},
-                message="Linking module to assets...",
-            ),
         ]
 
+        if self.cli_config.javascript_package_manager.name == "npm":
+            link_steps = [
+                FunctionStep(  # Create link to global folder
+                    func=self._run_script,
+                    args={"module_pkg": module_pkg},
+                    message="Linking module to global dist...",
+                ),
+                FunctionStep(  # Link the global folder to the assets folder.
+                    func=self._assets_link,
+                    args={"assets_pkg": assets_pkg, "module_pkg": module_pkg},
+                    message="Linking module to assets...",
+                ),
+            ]
+        else:
```

Comment:
In contrast to `npm`, `pnpm` supports linking via a single-step operation: https://pnpm.io/cli/link#pnpm-link-dir

Reply:
i think i don't understand, but does this mean we don't have to change anything in the react-invenio-forms package? since if `pnpm` is configured, `_assets_link_single` is taking care of linking the package and it doesn't care as what is in the `react-invenio-forms/package.json` file?